src/pages/Main/components/ProductLineup.scss

```
@@ -0,0 +1,72 @@
+@import '../../../styles/variables';
+
+.product-lineup {
+  position: relative;
+  margin-top: 32px;
+
+  @include flex-sort(space-between, center);
```

```suggestion
   @include flex-sort(space-between, center);
    position: relative;
   margin-top: 32px;
```

@include는 컨벤션 상 스타일의 가장 위에 위치 시켜주세요!

---

여기도 수정이 안되어있습니다!